checklists/alz_checklist.en.json

```diff
@@ -1763,28 +1708,6 @@
             "id": "E01.13",
             "severity": "Medium"
         },
-        {
-            "category": "Governance",
-            "subcategory": "Optimize your cloud investment",
-            "text": "Consider using automation tags to start/stop VM's in your environment to save on cost.",
```

Comment:
Is this being removed for duplicate, or for applicability?

I am good with removing this one, but we might want to include it in another checklist.

Reply:
Removing due to applicability